src/layout/PageTemplate.tsx

```diff
@@ -13,22 +12,28 @@ const PageTemplate: FC<{
 }> = ({ context, id, length, children, isLoading }) => {
   if (isLoading || !length) {
     return (
-      <div className="spinner-container">
+      <section className="spinner-container">
         <img src={spinner} alt="logo" width={120} height={50} />
-      </div>
+      </section>
     );
   }
 
   return (
-    <Fragment>
+    <section>
       <div className="page-header">
-        <h2 style={{ borderBottom: `1px solid ${colors.get(context)}` }}>
+        <h5
+          style={{
+            borderBottom: `1px solid ${
+              ContextObj[context as keyof typeof ContextObj].color
+            }`,
+          }}
+        >
           {length} {context.toLowerCase()}
           {length > 1 && "s"} for {id}
-        </h2>
+        </h5>
```

Comment:
From the design review - is it possible to expand the `for` to indicate the relationship. I recall from the meeting there is some subtlety about what data is being displayed - is it possible to clarify to the user by unpacking the context a bit more here?